toolkit/tools/scheduler/schedulerutils/buildworker.go

```diff
@@ -48,7 +49,7 @@ type BuildResult struct {
 	Err            error
 	LogFile        string
 	Node           *pkggraph.PkgNode
-	Skipped        bool
+	Ignored        bool
 	UsedCache      bool
 	WasDelta       bool
 }
```

Comment:
I've gotten confused by some of these, was thining maybe we should add a comment for each member of the req/res objects?
```suggestion
	Node           *pkggraph.PkgNode // Main node that was built
	Ignored        bool              // ???
	UsedCache      bool              // Used a cached component (expand on this later?)
	WasDelta       bool              // Used a pre-built published component
}
```